trame_rca/widgets/rca_helpers.py

```diff
@@ -0,0 +1,318 @@
+from __future__ import annotations
```

Comment:
This file should not be inside `trame_rca/widgets/`  but `trame_rca/utils.py` instead?

Reply:
There is no HTML widget in that file...

Reply:
Thanks for the feedback. I wasn't sure what was the convention for helper classes / methods. I moved everything to the `utils.py` file